### PR TITLE
Updated Unreal docs for v5.0

### PR DESCRIPTION
**Pull request by ddocs**

### CODE REVIEW

**Review by ektishafahwaz — CHANGES_REQUESTED** [review]
Please leave the following line in all functions to be the same.

Request->SetHeader(TEXT("User-Agent"), "X-MirageSDK-Agent");

**Review by ektishafahwaz — APPROVED** [review]
The modification is approved

File: docs/Game/Unreal/demo.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This Demo section walks through an example project to help get you started.
 
 ## View Demo Project
 
-Inside the Ankr/Mirage SDK click to open the ***unrealsdk.uproject***
+Inside the **Ankr SDK** click to open the **unrealsdk.uproject**
```
Comment (raproid) [inline docs/Game/Unreal/demo.md:10]: Inside the **Ankr SDK**, ?

File: docs/Game/Unreal/getting-started/00-before-you-start.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: 00 - Before you Start
 id: unreal-before
 ---
 
-## 👉🏻 Getting Started
+## 👉🏻 Getting started
 
 The Unreal Engine gets the back end client to connect and interact with the blockchain to send and retrieve data.
```
Comment (raproid) [inline docs/Game/Unreal/getting-started/00-before-you-start.md:8]: backend

Comment (raproid) [reply docs/Game/Unreal/getting-started/00-before-you-start.md:8]: what do you mean by "gets" — creates a client?

File: docs/Game/Unreal/getting-started/04-mint-nft.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ The response is a 'ticket'.
 The session saved during Init will be used to open metamask. Metamask will show a popup to sign or confirm the transaction for that ticket.
```
Comment (raproid) [inline docs/Game/Unreal/getting-started/04-mint-nft.md:27]: MetaMask?  You're writing it like this a lot.

And here you wrote "metamask" and Metamask".

Comment (raproid) [inline docs/Game/Unreal/getting-started/04-mint-nft.md:27]: is Init a function? If so, then `Init`?

File: docs/Game/Unreal/getting-started/04-mint-nft.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ The response is a 'ticket'.
...
```
Comment (raproid) [inline docs/Game/Unreal/getting-started/04-mint-nft.md:81]: is Init a function? If so, then `Init`?

File: docs/Game/Unreal/getting-started/04-mint-nft.md [hunk 5]
```diff
@@ -139,7 +139,7 @@ The response is a `ticket`.
...
```
Comment (raproid) [inline docs/Game/Unreal/getting-started/04-mint-nft.md:139]: is Init a function? If so, then `Init`?

Comment (raproid) [reply docs/Game/Unreal/getting-started/04-mint-nft.md:139]: and also MetaMask?

File: docs/Game/Unreal/getting-started/04-mint-nft.md [hunk 6]
```diff
@@ -241,7 +241,7 @@ The following body is sent to POST http://45.77.189.28:5000/result
 The response is a 'data' object containing the id of the character.
```
Comment (raproid) [inline docs/Game/Unreal/getting-started/04-mint-nft.md:241]: why 'data' and not data? why are you highlighting it in this way?

File: docs/Game/Unreal/getting-started/04-mint-nft.md [hunk 7]
```diff
@@ -290,7 +290,7 @@ The following body is sent to POST http://45.77.189.28:5000/send/transaction
 The session saved during Init will be used to open metamask. Metamask will show a popup to sign or confirm the transaction for that ticket.
```
Comment (raproid) [inline docs/Game/Unreal/getting-started/04-mint-nft.md:290]: `Init` and MetaMask?

File: docs/Game/Unreal/overview.md [hunk 8]
```diff
@@ -5,15 +5,15 @@ id: about-unreal
 
 The Ankr Unreal Game SDK brings powerful functionality to the Unreal Engine, making it easy for game developers and studios to give their games full Web3 capabilities- including integrating cryptocurrencies and non-fungible tokens (NFTs). These docs explain what Ankr’s Unreal Game SDK is, how to install it, and how to get started using it. Let’s dive in!
```
Comment (raproid) [inline docs/Game/Unreal/overview.md:6]: capabilities- including > capabilities, including ?

File: docs/Game/Unreal/setting-up.md [hunk 9]
```diff
@@ -7,72 +7,71 @@ Let's get started!
...
+3. Unzip the **AnkrSDK.zip** folder into the **Plugins** folder.
 
-4. Delete Binaries, Intermediate, Saved folders and Visual Studio Project file (.sln) from Unreal Project (If any)
+4. Delete **Binaries**, **Intermediate**, **Saved folders** and **Visual Studio Project file (.sln)** from Unreal Project (If any)
 
-#### Generate a Visual Studio Project
+## Generate a Visual Studio Project
 
-1. Generate a Visual Studio Project by Right-Clicking '[ProjectName].uproject' and selecting 'Generate Visual Studio project files.'
+1. Generate a **Visual Studio Project** by Right-Clicking **'[ProjectName].uproject'** and selecting **Generate Visual Studio project files**
```
Comment (raproid) [inline docs/Game/Unreal/setting-up.md:36]: by Right-Clicking > by right-clicking ?

File: docs/Game/Unreal/setting-up.md [hunk 10]
```diff
@@ -7,72 +7,71 @@ Let's get started!
...
 
-2. Open the generated 'Visual Studio solution file (.sln)' and check if the plugin in included in the solution explorer.
+2. Open the generated **Visual Studio solution file (.sln)** and check if the plugin in included in the solution explorer.
```
Comment (raproid) [inline docs/Game/Unreal/setting-up.md:38]: is included?

File: docs/Game/Unreal/setting-up.md [hunk 11]
```diff
@@ -7,72 +7,71 @@ Let's get started!
...
+2. Open your project folder **'[ProjectName].uproject'**
 
-3. Create a folder called 'Plugins'
+3. Create a folder called **'Plugins'**
 
-3. Unzip the 'AnkrSDK.zip' folder into the 'Plugins' folder.
+3. Unzip the **AnkrSDK.zip** folder into the **Plugins** folder.
 
-4. Delete Binaries, Intermediate, Saved folders and Visual Studio Project file (.sln) from Unreal Project (If any)
+4. Delete **Binaries**, **Intermediate**, **Saved folders** and **Visual Studio Project file (.sln)** from Unreal Project (If any)
```
Comment (raproid) [inline docs/Game/Unreal/setting-up.md:32]: (If any) > (if any).

File: docs/Game/Unreal/setting-up.md [hunk 12]
```diff
@@ -7,72 +7,71 @@ Let's get started!
...
-```
+      return ankrClient;
+   }
+   ```
 
-#### Add AnkrSDK to Build.cs
+## Add AnkrSDK to Build.cs
 
-Add "AnkrSDK" to your Unreal Project/Source/Unreal Project/Build.cs as shown below:
+Add **AnkrSDK** to your Unreal Project/Source/Unreal Project/Build.cs as shown below:
```
Comment (raproid) [inline docs/Game/Unreal/setting-up.md:69]: Unreal Project/Source/Unreal Project/Build.cs > `Unreal Project/Source/Unreal Project/Build.cs`

File: docs/Game/Unreal/setting-up.md [hunk 13]
```diff
@@ -7,72 +7,71 @@ Let's get started!
...
-```c++
+```cpp
 PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "AnkrSDK" });
 ```
 
-Click ***Edit->Project Settings->Maps and Modes*** and select your newly created or already created 'GameInstance' from the 'GameInstance Class' dropdown.
+Click **Edit > Project Settings >Maps and Modes** and select your newly created or already created **'GameInstance'** from the **'GameInstance Class'** dropdown.
 
-Now you can call all the functions in blueprint by getting 'GetGameInstance->GetMirageClient'.
+Now you can call all the functions in blueprint by getting **'GetGameInstance >GetAnkrClient'**.
```
Comment (raproid) [inline docs/Game/Unreal/setting-up.md:77]: **'GetGameInstance >GetAnkrClient'**. > **'GetGameInstance > GetAnkrClient'**.

Comment (raproid) [reply docs/Game/Unreal/setting-up.md:77]: space is missing

File: docs/Game/Unreal/getting-started/04-mint-nft.md (hunk 3 above)
Comment (ddocs) [reply docs/Game/Unreal/getting-started/04-mint-nft.md:27]: Done

Comment (ddocs) [reply docs/Game/Unreal/getting-started/04-mint-nft.md:27]: Done

File: docs/Game/Unreal/getting-started/04-mint-nft.md (hunk 4 above)
Comment (ddocs) [reply docs/Game/Unreal/getting-started/04-mint-nft.md:81]: Done

File: docs/Game/Unreal/getting-started/04-mint-nft.md (hunk 5 above)
Comment (ddocs) [reply docs/Game/Unreal/getting-started/04-mint-nft.md:139]: Done

File: docs/Game/Unreal/getting-started/04-mint-nft.md (hunk 7 above)
Comment (ddocs) [reply docs/Game/Unreal/getting-started/04-mint-nft.md:290]: done

File: docs/Game/Unreal/overview.md (hunk 8 above)
Comment (ddocs) [reply docs/Game/Unreal/overview.md:6]: done

File: docs/Game/Unreal/setting-up.md (hunk 9 above)
Comment (ddocs) [reply docs/Game/Unreal/setting-up.md:36]: done

File: docs/Game/Unreal/setting-up.md (hunk 10 above)
Comment (ddocs) [reply docs/Game/Unreal/setting-up.md:38]: Done

File: docs/Game/Unreal/setting-up.md (hunk 11 above)
Comment (ddocs) [reply docs/Game/Unreal/setting-up.md:32]: done

File: docs/Game/Unreal/setting-up.md (hunk 13 above)
Comment (ddocs) [reply docs/Game/Unreal/setting-up.md:77]: done

**Review by raproid — APPROVED** [review]

**Review by ektishafahwaz — COMMENTED** [review]
Good Work

**Pull request merged**